addons/web/static/src/core/datetime/datetime_picker_popover.xml

```diff
@@ -14,6 +14,17 @@
                     </button>
                 </t>
             </t>
+            <t t-set-slot="bottom_left">
+                <t t-if="isDateTimeRange">
+                    <button
+                        class="btn btn-secondary btn-sm h-100 w-100 w-md-auto d-flex align-items-center justify-content-center"
+                        tabindex="-1"
+                        t-on-click="props.close"
```

Comment:
Wait... Cancel and Apply both do the exact same thing? @bouvyd are you sure about this? I would expect Cancel to discard changes I made in the picker (and maybe to behave differently than pressing ESC or clicking outside).

Reply:
let's call it 'Close' instead of 'Cancel' and call it a day :D

Reply:
Changes has been done.